test/remote/gateways/remote_nuvei_test.rb

```diff
@@ -74,6 +97,22 @@ def test_successful_purchase
     assert_match 'SUCCESS', response.params['status']
   end
 
+  def test_successful_purchase_with_3ds_frictionless
+    response = @gateway.purchase(@three_ds_amount, @frictionless_credit_card, @options.merge(@three_ds_options))
+    assert_success response
+    assert_not_nil response.params[:transactionId]
+    assert_match 'SUCCESS', response.params['status']
+    assert_match 'APPROVED', response.message
+  end
+
+  def test_successful_purchase_with_3ds_challenge
+    response = @gateway.purchase(@three_ds_amount, @challenge_credit_card, @options.merge(@three_ds_options))
+    assert_success response
+    assert_not_nil response.params[:transactionId]
+    assert_match 'SUCCESS', response.params['status']
+    assert_match 'REDIRECT', response.message
+  end
+
```

Comment:
💬 Comment

Maybe is a good idea to add a test that covers the "failure" scenario where the cc is not enrolled